Build the rust bridge manually instead of using bazel. See APKBUILD for
reasoning.
--- anki-2.1.49/defs.bzl	2021-10-26 02:53:24.000000000 +0200
+++ anki-2.1.49_rust_output/defs.bzl	2022-01-03 16:16:45.645536303 +0100
@@ -1,6 +1,5 @@
 load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 load("@bazel_skylib//lib:versions.bzl", "versions")
-load("@rules_rust//rust:repositories.bzl", "rust_repositories")
 load("@ankidesktop//cargo:crates.bzl", "raze_fetch_remote_crates")
 load(":python.bzl", "setup_local_python")
 load("//proto:protobuf.bzl", "setup_protobuf_binary")
@@ -18,13 +18,6 @@ def setup_deps():
 
     versions.check(minimum_bazel_version = "3.7.0")
 
-    rust_repositories(
-        edition = "2018",
-        include_rustc_srcs = False,
-    )
-
-    raze_fetch_remote_crates()
-
     setup_local_python(name = "python")
 
     setup_protobuf_binary(name = "com_google_protobuf")
